### PR TITLE
docs: add version table

### PR DESCRIPTION
**Pull request by nornagon**
#### Description of Change
This info exists at http://atom.io/download/electron/index.json, but this makes it more discoverable.

#### Checklist

- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [ ] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: no-notes

### CODE REVIEW

**Comment by nornagon** [comment]
draft because i want to include a script that updates this automatically.

File: docs/versions.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+| Electron Version | Chromium Version | Node Version | `NODE_MODULE_VERSION` |
+|==================|==================|==============|=======================|
+| 2.x              | 61.0.3163.100    | 8.9.3        | 57                    |
+| 3.x              | 66.0.3359.181    | 10.2.0       | 64                    |
+| 4.x              | 69.0.3497.106    | 10.11.0      | 69                    |
+| 5.x              | 73.0.3683.27     | 12.0.0       | 68                    |
```
Comment (codebytere) [inline docs/versions.md]: let's also put v8 here? see https://github.com/electron/node#branching-strategy

**Comment by Kilian** [comment]
We also have this: https://github.com/electron/releases that has a script as well as a table.

**Comment by nornagon** [comment]
yeah this probably shouldn't be here

**Pull request closed without merging**